### PR TITLE
Display user data

### PR DESCRIPTION
**Pull request by collinkallery**
# PR Description
Please include a summary of the change:
- Accurately displaying data for the user and their friends up to 01/22/2020. 
- Notes / checklist for proceeding through the next 7 days of the project. 

Commit Emojis:

:crystal_ball: JS modifications/updates.
:memo: Pseudo-coding and planning

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
- The only new feature here is accurate fetching and displaying of fetched data. 

Note any questions or struggles as well.

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -275,94 +190,236 @@ let updateTrendingStepDays = (user) => {
...
+  //Refactor this into a forEach
+  for (var i = 0; i < $('.daily-oz').length; i++) {
+    $('.daily-oz')[i].text(user.addDailyOunces(Object.keys(sortedHydrationDataByDate[i])[0]))
+  };
+  //splice sortedHydrationDataByDate to get most recent 7 entires
+  //loop through the first 7 entries and populate the oz per day
+}
 
-// user.findFriendsTotalStepsForWeek(userRepository.users, todayDate);
+// 1 WEEK LEFT CHECKLIST:
```
Comment (collinkallery) [inline src/scripts.js:304]: Notes here for proceeding throughout the week. Please refer to this as we complete items and check them off as we go, and add any necessary tasks or to-dos.

File: src/scripts.js [hunk 2]
```diff
@@ -39,37 +36,38 @@ hydrationData = fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydratio
...
+    userRepository = new UserRepository(userData, sleepData, activityData, hydrationData);
     instantiateAllUsers();
     instantiateAllUsersActivity();
     instantiateAllUsersHydration();
     instantiateAllUsersSleep();
   })
   .then(() => {
     let user = userRepository.users[Math.floor(Math.random() * userRepository.users.length)]
-    let todayDate = "2019/09/22";
+    let todayDate = "2020/01/22";
```
Comment (Karlfunhouse) [inline src/scripts.js:57]: updating current date from dataset.  Will go back to make it dynamic once we can start posting new data into the database.

**Comment by Melizzo** [comment]
Was present for 70% of coding - all code looks good and appears to have no conflicts.

File: src/scripts.js [hunk 3]
```diff
@@ -275,94 +190,236 @@ let updateTrendingStepDays = (user) => {
...
+//     body: JSON.stringify({
+//         "userId": `${user-id-input}`,
+//         "date": `${date-input}`,
+//         "hoursSlept": `${hours-slept-input}`,
+//         "sleepQuality": `${sleep-quality-input}`
+//     })
+//   })
+// };
 //
-// user.friendsActivityRecords.forEach(friend => {
```
Comment (Karlfunhouse) [inline src/scripts.js:362]: refactored this into jQUery using .append()

**Pull request merged**